### DISCUSSION TITLE
duck --edit macos doesn't exit

### DISCUSSION THREAD

**Discussion by wellithy** [iterate-ch/cyberduck#15668] [Q&A] [answered]
What is the correct way to use `duck --edit` ?

*Details*
I'm using CLI on macOS
```
$ duck --version
Cyberduck 8.7.1 (40770)
$ uname 
Darwin
```

When I edit a fie, it saves it remotely correctly.  However, when I quit the editor, I have to force the command to exit `Ctrl-C`
```
$ duck --edit ...
Upload complete ...
^C+ exit
```

And in some cases the process doesn't completely exit
```
$ ps
 4059 ttys000    0:07.63 duck --edit ...
```
and I have to `kill -9`.

Thanks

**Comment by dkocher** [comment, marked as answer]
Our [documentation](https://docs.duck.sh/cli/#edit-with-edit) is too sparse and should be extended. It is expected that the command blocks and watches for changes from the editor. The process will only exit with `Ctrl-c` of when the editor application is quit. I don't see a reason why the process would still hang but will try to reproduce.

**Reply by wellithy** [reply]
Thanks @dkocher for the quick response.
I'll update this thread if/when I can systematically reproduce the scenario(s) that i had to use `kill`.
